plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -417,6 +418,21 @@ instance HasConstantIn DefaultUni term => ReadKnownIn DefaultUni term Word8 wher
                _       -> throwing_ _EvaluationFailure
     {-# INLINE readKnown #-}
 
+-- | This allows us to transparently use values of type LiteralByteSize as
+-- built-in Integers but with a different size measure.
+instance KnownTypeAst tyname DefaultUni LiteralByteSize where
+    toTypeAst _ = toTypeAst $ Proxy @Integer
+
+-- See Note [Integral types as Integer].
+instance HasConstantIn DefaultUni term => MakeKnownIn DefaultUni term LiteralByteSize where
+    makeKnown = makeKnown . unLiteralByteSize
+    {-# INLINE makeKnown #-}
+
+instance HasConstantIn DefaultUni term => ReadKnownIn DefaultUni term LiteralByteSize where
+    readKnown term = LiteralByteSize <$> readKnown term
+    {-# INLINE readKnown #-}
+
+
```

Comment:
Nit: two newlines suck.

Reply:
That was probably just accidental, but sometimes I find it useful to have a bit more space to break the code into blocks of related functionality. Maybe in that case I should use a comment rather than blank lines.

Reply:
I usually use a comment indeed. Rarely I use three newline. Two newlines look like accidental noise to me (not enough signal like with three lines), but that's clearly just me being crazy.